news/4.35/platform.html

```diff
@@ -97,6 +97,32 @@ <h2>General Updates </h2>
         </p>
       </td>
   </tr>
+  <tr id="edgeBrowserDefault"> <!-- https://github.com/eclipse-platform/eclipse.platform.swt/issues/1466 -->
+    <td class="title">Edge as Default Browser on Windows</td>
+    <td class="content">
+      <p>
+        Edge replaces Internet Explorer as the default browser instantiated by SWT. In consequence, every SWT- and
+        Eclipse-based product will by default now create instances of the <code>Browser</code> class based
+        on Edge when running on Windows. As some examples, this affects the Welcome page and the help in an Eclipse product,
+        and the Javadoc view and the Java element tooltips in a Java IDE product.
+      </p>
+      </p>
+        <b>Feedback:</b> In case you experience any issues or limitations with the Edge browser implementation, please report
+        them back in the umbrella issue (<a href="https://github.com/eclipse-platform/eclipse.platform.swt/issues/1466">#1466</a>).
+      </p>
+      <p>
+        In case you want to or need to still use Internet Explorer in your Eclipse product, you can:
+        <ul>
+          <li>Switch to Internet Explorer for a complete product by setting the VM argument (for example in your
+            <i>eclipse.ini</i>: <code>-Dorg.eclipse.swt.browser.DefaultType=IE</code>
+          </li>
+          <li>Programmatically create a single browser instance with Internet Explorer by passing the flag
```

Comment:
`Browser` also mentions a `BrowserInitializer` via a fragment:
https://github.com/eclipse-platform/eclipse.platform.swt/blob/9d3673c858777004c479d05cdd8277d80ccfe95a/bundles/org.eclipse.swt/Eclipse%20SWT%20Browser/common/org/eclipse/swt/browser/Browser.java#L157-L170
Not sure if we should mention this approach here.

Reply:
Interesting. I was not aware of that possibility. Thank you for the pointer! I added an according explanation to the news.